src/AppCommon/Commands/AzureServiceBusCommand.cs

```diff
@@ -116,13 +111,20 @@ protected override async Task<QueueDetails> GetData(CancellationToken cancellati
         }
     }
 
-    protected override Task<EnvironmentDetails> GetEnvironment(CancellationToken cancellationToken = default)
+    protected override async Task<EnvironmentDetails> GetEnvironment(CancellationToken cancellationToken = default)
     {
-        return Task.FromResult(new EnvironmentDetails
+        Out.WriteLine($"Getting data from {azure.FullyQualifiedNamespace}...");
```

Comment:
For some reason Azure ServiceBus was not collecting the queue names as part of this method call.
This is the only command that is different.
So we modified this code to line up with the other commands, so we can use this method to get the list of queues only.